cluster/addons/prometheus/prometheus.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1beta1
+kind: Pod
+id: kube-prometheus
+desiredState:
+  manifest:
+    version: v1beta1
+    containers:
+      - name: kube-promdash
+        image: prom/promdash
+        ports:
+          - containerPort: 3000
+            hostPort: 3000
+        env:
+          - name: "DATABASE_URL"
+            value: "sqlite3:/promdash/file.sqlite3"
```

Comment:
Can we comment out a block for folks to enable volume mounts for persistence. 

/cc @markturansky